lib/ansible/plugins/connection/ssh.py

```diff
@@ -185,6 +185,14 @@
         env: [{name: ANSIBLE_SCP_IF_SSH}]
         ini:
         - {key: scp_if_ssh, section: ssh_connection}
+      use_tty:
+        default: True
+        description: add -tt to ssh commands to force tty allocation
+        env: [{name: ANSIBLE_SSH_USETTY}]
+        ini:
+        - {key: usetty, section: ssh_connection}
+        type: boolean
+        yaml: {key: ssh_connection.usetty}
```

Comment:
YAMLis not in use yet, but in the plan it would be under `connection.ssh.usetty`
Adding connection var is also useful for making it 'per host/group'